packages/plugin-core/src/commands/pods/ImportObsidianCommand.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  getAllImportPods,
+  MarkdownImportPod,
+  podClassEntryToPodItemV4,
+} from "@dendronhq/pods-core";
+import vscode from "vscode";
+import { PickerUtilsV2 } from "../../components/lookup/utils";
+import { DENDRON_COMMANDS } from "../../constants";
+import { PodQuickPickItemV4 } from "../../utils/pods";
+import { CommandOpts, ImportPodCommand } from "../ImportPod";
+
+/**
+ * Convenience command that uses the same flow as {@link ImportPodCommand} for
+ * Markdown Pod but simplifies the steps by not requiring the user to fill out a
+ * config.yml file.
+ */
+export class ImportObsidianCommand extends ImportPodCommand {
```

Comment:
this might be a markdown import pod issue but right now, we don't show a progress bar when importing which makes it hard to tell if import is still going or something has crashed. can we show a window.progress notification during the import?

Reply:
It looks like we already do this in Import Pod:

https://github.com/dendronhq/dendron/blob/08e8320dc88ef52c1c49e76ace843f8a50642225/packages/plugin-core/src/commands/ImportPod.ts#L129

https://github.com/dendronhq/dendron/blob/08e8320dc88ef52c1c49e76ace843f8a50642225/packages/plugin-core/src/commands/ImportPod.ts#L172